runMultiQC.nf

```diff
@@ -50,28 +50,28 @@ process GetVersionAll {
   publishDir directoryMap.multiQC, mode: 'link'
 
   input:
-    file(versions) from Channel.fromPath("${directoryMap.version}/*").collect()
+    file(versions) from Channel.fromPath("${directoryMap.version}/*").collect().ifEmpty(file ("empty"))
 
   output:
-  file ("tool_versions_mqc.yaml") into versionsForMultiQC
+    file ("tool_versions_mqc.yaml") into versionsForMultiQC
 
   when: !params.noReports
 
   script:
   """
-  bcftools version > v_bcftools.txt
+  bcftools version > v_bcftools.txt 2>&1 || true
   bwa &> v_bwa.txt 2>&1 || true
-  configManta.py --version > v_manta.txt
-  configureStrelkaGermlineWorkflow.py --version > v_strelka.txt
-  echo "${params.version}" &> v_sarek.txt
-  echo "${workflow.nextflow.version}" &> v_nextflow.txt
-  fastqc -v > v_fastqc.txt
-  freebayes --version > v_freebayes.txt
-  gatk ApplyBQSR --help 2>&1 | grep Version: > v_gatk.txt
-  multiqc --version &> v_multiqc.txt
-  qualimap --version &> v_qualimap.txt
-  samtools --version &> v_samtools.txt
-  vcftools --version > v_vcftools.txt
+  configManta.py --version > v_manta.txt 2>&1 || true
```

Comment:
Will be much harder to spot errors with MultiQC if we ignore all potential version fetching errors, no?

Reply:
I guess it doesn't really matter, because, if a tool is failing, it will be seen when we're really using the tool, not when we want to check its version.
Worst case scenario here, we won't have it display properly.